pkg/status/templates/rendererrors.tmpl

```diff
@@ -0,0 +1,11 @@
+{{/*
+NOTE: Changes made to this template should be reflected on the following templates, if applicable:
+* cmd/agent/gui/views/templates/generalStatus.tmpl
```

Comment:
We also need to update the web GUI.

Reply:
@hush-hush 
I think, for the web GUI, a template file doesn't need because an error is already rendered like this.
https://a.cl.ly/E0uRwk2k (Modified the code to cause this error intentionally)
If [fillTemplate](https://github.com/DataDog/datadog-agent/blob/1ba2e4d7d04b533d229dc9ff88c3ce2c28ecffc9/cmd/agent/gui/render.go#L95-L108) returns an error, it will appear in the web GUI.

What do you think?

Reply:
I didn't knew, :+1: